client/snap_op.go

```diff
@@ -226,15 +226,43 @@ func (client *Client) InstallPath(path, name string, options *SnapOptions) (chan
 		SnapOptions: options,
 	}
 
+	return client.sendLocalSnaps([]string{path}, []*os.File{f}, action)
+}
+
+// InstallPathMany sideloads the snaps with the given paths,
+// returning the UUID of the background operation upon success.
```

Comment:
A bit weird we called it UUID (I see it's taken from the existing comment) and not just change id...